# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcTCPSocketChild.cpp
# Commit: f5353ea9f1df
# Full Hash: f5353ea9f1dfe15a65076bc7aa272d688ebdb8ca
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Add a proxy policy argument to NrSocketProxyConfig, and always set this config on the NrIceCtx. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45101
# ==============================================================================

diff -r ed1ae6a85255 -r f5353ea9f1df media/mtransport/ipc/WebrtcTCPSocketChild.cpp
--- a/media/mtransport/ipc/WebrtcTCPSocketChild.cpp	Tue Sep 17 18:17:13 2019 +0000
+++ b/media/mtransport/ipc/WebrtcTCPSocketChild.cpp	Tue Sep 17 18:17:14 2019 +0000
@@ -63,10 +63,10 @@
   LOG(("WebrtcTCPSocketChild::~WebrtcTCPSocketChild %p\n", this));
 }
 
-void WebrtcTCPSocketChild::AsyncOpen(const nsCString& aHost, const int& aPort,
-                                     const net::LoadInfoArgs& aArgs,
-                                     const nsCString& aAlpn,
-                                     const dom::TabId& aTabId) {
+void WebrtcTCPSocketChild::AsyncOpen(
+    const nsCString& aHost, const int& aPort, const net::LoadInfoArgs& aArgs,
+    const nsCString& aAlpn, const dom::TabId& aTabId,
+    NrSocketProxyConfig::ProxyPolicy aProxyPolicy) {
   LOG(("WebrtcTCPSocketChild::AsyncOpen %p %s:%d\n", this, aHost.get(), aPort));
 
   MOZ_ASSERT(NS_IsMainThread(), "not main thread");
@@ -85,7 +85,7 @@
                                                                         aTabId);
   }
 
-  SendAsyncOpen(aHost, aPort, aArgs, aAlpn);
+  SendAsyncOpen(aHost, aPort, aArgs, aAlpn, aProxyPolicy);
 }
 
 }  // namespace net